models/twilio_message_enhanced.sql

```diff
@@ -80,8 +80,8 @@ final as (
 
     select
         union_messages.message_id,
-        union_messages.messaging_service_id,
-        union_messages.date_sent,
+        union_messages.messaging_service_id, 
```

Comment:
This is null for all our records. Is that intentional?

Additionally, should we be joining in the messaging_service staging model here to add additional information to the message enhanced model?

Reply:
Yeah, it looks like it's null in our message object but that makes sense, the record in our messaging_service doesn't look to be used for Messages (rather, for Conversations) so I assume it's different.

Reply:
And good point, yes brought in some relevant fields from that staging model!

Reply:
What do you mean by 

> the record in our messaging_service doesn't look to be used for Messages (rather, for Conversations) so I assume it's different

Does this mean the messaging_service is for a different component and shouldn't be brought into the model? Also if we do end up bringing it into the model, be sure to add a config to turn it off if users don't have the messaging_service component.

Reply:
so based on our [ERD](https://docs.google.com/presentation/d/1ZQ3qkzxXX4fFjv1MTLepwsSC6RvEoqVGMufUP52i4Eo/edit#slide=id.g83364f78b7_1_0) it looks like messaging_service can be used for either Message or Conversation, so for our available data it looks like it is for the latter. But we should still keep it in the message model.